lib/livebook_web/live/session_live/shortcuts_component.ex

```diff
@@ -18,11 +18,10 @@ defmodule LivebookWeb.SessionLive.ShortcutsComponent do
         desc: "Show signature help"
       },
       %{
-        seq: ["ctrl", "i"],
-        seq_mac: ["⌘", "i"],
+        seq: ["alt", "i"],
+        seq_mac: ["⌥", "i"],
         press_all: true,
-        desc: "Show identifier documentation",
-        basic: true
+        desc: "Show identifier documentation"
```

Comment:
`alt/option + i` seems safe to use, so I think we can try that. Note that the default `ctrl+k ctrl+i` still works, it's just not particularly handy.

However I don't think we need it in the basic view, since the docs are triggered by hover.

Reply:
Nah, I would remove it altogether, less keybindings is easier to maintain compatibility long term. Imagine then in a future version Monaco or the OS adds something to `alt+i`. So I would really stick with Monaco defaults. :)